### PR TITLE
chore(react-scheduler): remove excess getters

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**

### CODE REVIEW

File: packages/dx-scheduler-core/src/plugins/all-day-panel/helpers.js [hunk 1]
```diff
@@ -1,61 +1,18 @@
...
-const CELL_GAP = 0.32;
-const CELL_BOUND_OFFSET_PX = 2;
-
 export const allDayPredicate = appointment => (
   appointment.end.diff(appointment.start, 'hours') > 23
   || !!appointment.allDay
 );
 
-const getCellRect = (date, dayScale, cellElements, takePrev) => {
+export const getAllDayCellByDate = (viewCellsData, date, takePrev) => {
```
Comment (SergeyAlexeev) [inline packages/dx-scheduler-core/src/plugins/all-day-panel/helpers.js]: The `getAllDayCellByDate` name sounds strange. I assume the function returns a cell, however it returns a cell index.

Comment (MaximKudriavtsev) [reply packages/dx-scheduler-core/src/plugins/all-day-panel/helpers.js]: Can we rename this to `getAllDayCellIndexByDate` ?

File: packages/dx-scheduler-core/src/plugins/horizontal-rect/helpers.test.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+          viewCellsData,
+          cellElements,
+          multiline: true,
+        },
+      );
+
+      expect(top).toBe(132);
+      expect(left).toBe(12);
+      expect(height).toBe(68);
+      expect(width).toBe(98); // !!!!!
```
Comment (SergeyAlexeev) [inline packages/dx-scheduler-core/src/plugins/horizontal-rect/helpers.test.js]: `// !!!!!`

Is it required?

File: packages/dx-scheduler-core/src/plugins/month-view/helpers.js [hunk 3]
```diff
@@ -34,53 +31,14 @@ export const sliceAppointmentByWeek = (timeBounds, appointment, step) => {
   return pieces;
 };
 
-const getCellRect = (date, monthCells, cellElements, takePrev) => {
-  const startViewDate = moment(monthCells[0][0].value);
+export const getMonthCellByDate = (viewCellsData, date, takePrev = false) => {
```
Comment (SergeyAlexeev) [inline packages/dx-scheduler-core/src/plugins/month-view/helpers.js]: The same about a function name.

File: packages/dx-scheduler-core/src/plugins/all-day-panel/helpers.js (hunk 1 above)
Comment (SergeyAlexeev) [reply packages/dx-scheduler-core/src/plugins/all-day-panel/helpers.js]: I think so.

**Pull request merged**